src/content/drafts/redesigning-eslint.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: Redesigning ESLint
+teaser: "A brief insight into ESLint's recent brand refresh and website redesign."
+tags:
+  - brand
+  - design
+authors:
+  - haydenbleasel
+categories:
+  - Storytime
+---
+
+Hi! My name's Hayden. I'm a designer and developer and, until recently, I ran a little agency called [Jellypepper](https://jellypepper.com).
+
+We were fortunate enough to work with the ESLint team on their new brand and website. I'm a huge fan of ESLint and use it every day, even going to the effort of writing my own [600-line ESLint config](https://github.com/haydenbleasel/harmony) to use across all my React, Next.js and Expo projects.
+
+[Nicholas](https://github.com/nzakas) originally reached out to us on Twitter to kick off the largest project the ESLint team has taken on in its nine years of existence. He wanted to create a stronger brand identity for the ESLint team and its community, and make the website more functional, easier to navigate, and more useful to everyone from new users to established users. 
+
+> Hi Hayden, I wanted to say thanks for the donation to ESLint. As it so happens, the project is looking for a bit of a rebrand (keep the logo, change everything else) and website redesign, so I was wondering if you’d be interested. It just seemed like a potentially good match.
+
+This was a bit of a dream project for me 😅 So, needless to say, we were totally in.
+
+## The Brand
+
+The ESLint Brand is an interesting one - without too much official visual design, the team have managed to create a brand that is unique and widely recognized by the developer community.
+
+### The Logo
+
+The logomark is at the core of ESLint's brand identity and one of the most identifiable pieces. It’s a connection to their ecosystem - you can find it everywhere, from IDE marketplaces to corporate sponsorship pages across the web. It carries with it the expectation of a community of developers who are passionate about the code they write.
+
+While we kept the icon (given its widespread usage in websites, IDEs, platforms and projects), we wanted to make it that little bit more refined, modern and flexible. By bringing it together our new primary typeface, Space Grotesk (more on this below), we created a new ESLint logomark. Due to its simplicity, it works in a variety of colours and spaces.
+
+![ESLint Logo Mockup](/src/assets/images/blog/2022/logo-mockup.png)
```

Comment:
The images are not rendering in the preview. Can you try with path starting with `/assets` instead of `/src/assets` ?

Reply:
Done! Sorry, the `/src` is what helped my preview render properly while I was drafting.